vmdb/app/models/provider.rb

```diff
@@ -8,6 +8,9 @@ class Provider < ActiveRecord::Base
   belongs_to :zone
   has_many :managers, :class_name => "ExtManagementSystem"
 
+  default_value_for :verify_ssl, OpenSSL::SSL::VERIFY_PEER
+  validates :verify_ssl, :inclusion => {:in => [0, 1]}
```

Comment:
Should this be `:in => [OpenSSL::SSL::VERIFY_NONE, OpenSSL::SSL::VERIFY_PEER]` ?